lapack/netlib/lapack.go

```diff
@@ -532,6 +532,134 @@ func (impl Implementation) Dlange(norm lapack.MatrixNorm, m, n int, a []float64,
 	return lapacke.Dlange(byte(norm), m, n, a, lda, work)
 }
 
+// Dlansb returns the given norm of an n×n symmetric band matrix with kd
+// super-diagonals.
+//
+// When norm is lapack.MaxColumnSum or lapack.MaxRowSum, the length of work must
+// be at least n.
+func (impl Implementation) Dlansb(norm lapack.MatrixNorm, uplo blas.Uplo, n, kd int, ab []float64, ldab int, work []float64) float64 {
```

Comment:
Just wondering, can we do something like

```
func (impl Implementation) Dlansb(norm lapack.MatrixNorm, uplo blas.Uplo, n, kd int, ab []float64, ldab int, work []float64) float64 {
    return gonum.Dlansb(norm, uplo, n, kd, ab, ldab, work)
}
```

I suspect this will pull in the entirety of the `gonum.Implementation` which is not what we want. I *think* this happens though I am not sure.

Reply:
As discussed in https://github.com/gonum/gonum/issues/1428, I've abandoned this approach and reverted the addition.